src/libraries/System.Formats.Tar/tests/TarTestsBase.cs

```diff
@@ -58,6 +59,8 @@ public abstract partial class TarTestsBase : FileCleanupTestBase
 
         protected const string TestGName = "group";
         protected const string TestUName = "user";
+        protected readonly string TestLongGName = new string('�', 33);
+        protected readonly string TestLongUName = new string('�', 100);
```

Comment:
I think there was a policy in the repo about hardcoding non-ASCII character (ñ and ü are extended ASCII, though). @danmoseley?

Regardless, is there a reason to use them in the `WriteEntry_CopyArchive` scenario?
```suggestion
        protected readonly string TestLongGName = new string('a', 33);
        protected readonly string TestLongUName = new string('a', 100);
```

Reply:
Ah yeah, the escaped utf8 values.

- `ä` is `\u00e4`
- `ñ` is `\u00f1`

Found here: http://daniel-hug.github.io/characters/

Reply:
The question remains, why not just use an ASCII character?

Reply:
Yeah, https://github.com/dotnet/runtime/blob/7a45201181d37329b7ab0bee541ed7c42b5d94b6/docs/coding-guidelines/coding-style.md?plain=1#L29
It's because sometimes editors garble them, or it causes notepad to add a BOM (?) 
Don't reset validation just for this -- plenty of places we're not doing this already.

Reply:
> The question remains, why not just use an ASCII character?

We want to make sure we are able to encode/decode UTF8 chars correctly. And since ASCII is a subset of UTF8, it's covered.